src/routes/+page.svelte

```diff
@@ -294,4 +335,13 @@
         width: 100%;
         height: auto;
     }
+
+    .banner {
+        background-image: url("/banner.webp");
```

Comment:
We're going to need to export that coin image separately. In wider screen it gets disproportionately big as the image stretches, also we'll need to restrict the max width between the coin and the text, so we don't end up with the huge gap between the two elements.

The coin image it's already provided separately from the background on Figma 

![image](https://github.com/user-attachments/assets/e5e90768-9015-43d2-aa10-0b1fc3e47d4c)